libs/books/data-access/src/lib/+state/reading-list.selectors.ts

```diff
@@ -5,7 +5,7 @@ import {
   READING_LIST_FEATURE_KEY,
   readingListAdapter,
   ReadingListPartialState,
-  State
+  State,
```

Comment:
Remove extra , from her and check for all files where you have made such mistake.